### PR TITLE
React/week1

### PR DESCRIPTION
**Pull request by simonreddy2001**
waiting for feedback

### CODE REVIEW

File: react/week1/todo-list/src/index.js [hunk 1]
```diff
@@ -1,18 +1,15 @@
...
-  return <li>{props.todo}, {props.deadline}</li>
-}
 ReactDOM.render(
   <>
     <h1>Todo List</h1>
     <ul>
-      <Todo todo="Get out of bed" deadline={"Wed Sep 13 2017"} />
-      <Todo todo="Brush teeth" deadline={"Thu Sep 14 2017"} />
-      <Todo todo="Eat breakfast" deadline={"Fri Sep 15 2017"} />
+      {todos.map((aTodo) => <Todo todo={aTodo.todo} deadline={aTodo.deadline} />)}
```
Comment (rufaidaju) [inline react/week1/todo-list/src/index.js:12]: Here the console is showing a warning that says Each child in a list should have a unique "key" prop.
you can avoid the problem by providing a unique key that has the value of todo.id 
<Todo todo={aTodo.todo} deadline={aTodo.deadline}  key={aTodo.id}/>
 It is also good to have [ Naming Conventions](https://github.com/HackYourFuture/fundamentals/blob/master/fundamentals/naming_conventions.md) to name your variables

**Comment by rufaidaju** [comment]
Hi Simon
Good job with the homework.
I have small things  to mention:
Name your components (file)
To figure which component has a bug, it’s important to always give your component a name.  I would name App.js to be 
Todo.js
 It is also better especially when you have a big project, it helps you to organize your project, and usually, when you have many components we add them to one folder called components.
Good luck :)

File: react/week1/todo-list/src/index.js (hunk 1 above)
Comment (simonreddy2001) [reply react/week1/todo-list/src/index.js:12]: sure .. thanks

**Comment by simonreddy2001** [comment]
thank you for feedback.. fixed according to it..

**Pull request merged**